x-pack/plugins/observability_solution/observability_ai_assistant/public/hooks/use_once.ts

```diff
@@ -9,6 +9,9 @@ import { useRef } from 'react';
 
 export function useOnce<T>(variable: T): T {
   const ref = useRef(variable);
+  if (!ref.current && variable) {
```

Comment:
NOooOOO!!! #2

Reply:
Reverted in https://github.com/elastic/kibana/pull/194552/commits/911c57e4768d24833c236d942d6c860f4fed41f9